### DISCUSSION TITLE
ESP32 C6 + ZigBee (ZigBee2MQTT) support

### DISCUSSION THREAD

**Discussion by LsHallo** [jomjol/AI-on-the-edge-device#2515] [General] [answered]
### The Feature

I would like to use ZigBee to transmit the information. Since the WiFi signal regularly drops for my current ESP32CAM module. It's just not reliable.

Since the C6 has WiFi + ZigBee you could trigger the settings menu via an mqtt message and it would connect to your wifi (wifi and zigbee concurrency is tricky i heard).  After configuration is done you then reboot and switch to zigbee mode again.

I have tried to create a ZigBee device with an ESP32H2 with little success. The properties and functions for ZigBee are not properly documented (or I can't find them).
I also don't know if there are ESP32C6 versions with enough RAM available. This is just an idea.

Edit: I've come to realize an ESP32C6 might not support a camera. If that's the case I would like to adjust the feature request to inter IC communication so that a ZigBee ESP32 can take on the communication rule. I even will take a look into that myself.

**Comment by caco3** [comment, marked as answer]
If it is acceptable for you to have 2 devices, the easiest ist to link the UART and use the output of every round, eg. `[23d19h49m47s] 2023-07-09T17:44:47 <INF> [POSTPROC] main: Raw: 580.06912, Value: 580.06912, Status: no error`.
This can then be used to do whatever you want on your 2nd device.
